store/sqlstore/channel_store.go

```diff
@@ -37,6 +38,203 @@ type SqlChannelStore struct {
 	metrics einterfaces.MetricsInterface
 }
 
+type channelMember struct {
+	ChannelId    string
+	UserId       string
+	Roles        string
+	LastViewedAt int64
+	MsgCount     int64
+	MentionCount int64
+	NotifyProps  model.StringMap
+	LastUpdateAt int64
+	SchemeUser   bool
+	SchemeAdmin  bool
+}
+
+func NewChannelMemberFromModel(cm *model.ChannelMember) *channelMember {
+	return &channelMember{
+		ChannelId:    cm.ChannelId,
+		UserId:       cm.UserId,
+		Roles:        cm.ExplicitRoles, // FIXME: Do we need to do this or anything else, since we are *not* Lazy Migrating.
```

Comment:
Not here. Once a row has been migrated the `Roles` field in the database should only contain the explicit ones, so that the Admin/User ones from the Scheme stay automatically in sync with scheme changes.